cleanlab/multiannotator.py

```diff
@@ -0,0 +1,829 @@
+import numpy as np
+import pandas as pd
+from typing import List, Union, Tuple
+from cleanlab.rank import get_label_quality_scores
+from cleanlab.internal.util import get_num_classes
+from cleanlab.internal.label_quality_utils import _get_label_quality_scores_with_NA
+from cleanlab.dataset import _get_worst_class
+import warnings
+
+
+def convert_long_to_wide_dataset(
+    labels_multiannotator_long: pd.DataFrame,
+) -> pd.DataFrame:
+    """Converts a wide format dataset to long format which is suitable for passing into
+    :py:func:`get_label_quality_multiannotator <cleanlab.multiannotator.get_label_quality_multiannotator>`.
+    Dataframe must contain three columns named:
+    1. ``task`` representing each example labeled by the annotators
+    2. ``annotator`` representing each annotator
+    3. ``label`` representing the label given by an annotator for the corresponding task
+
+    Parameters
+    ----------
+    labels_multiannotator_long : pd.DataFrame
+        pandas DataFrame in long format with three columns named ``task``, ``annotator`` and ``label``
+    """
```

Comment:
could clarify this:

Returns
--------

labels_multiannotator_wide : pd.DataFrame
   Object of the proper format to be passed as \`\`labels_multiannotator\`\` for the other \`\`cleanlab.multiannotator\`\` functions.

Reply:
Left the returns section out by accident, good catch!